### DISCUSSION TITLE
Changes to sqlcmd releases

### DISCUSSION THREAD

**Discussion by shueybubbles** [microsoft/go-sqlcmd#537] [Announcements]
For the next few months, we are unable to publish new sqlcmd packages to the linux package managers or docker repository. We are trying to get appropriate resources engaged in making this process work again after new secure publishing practices were instituted at Microsoft. We apologize for the inconvenience!

Another change starting with [v1.7.0](https://github.com/microsoft/go-sqlcmd/releases/v1.7.0) is we have removed the version number from the file names. Thus you could start using a fixed url to `latest` and always get the newest bits.

Please try out v1.7.0. After it's in pre-release for a week or so with no negative bug reports I will mark it `latest`. 

thx again for using our tools and we value your feedback!

**Comment by rglidden** [comment]
@shueybubbles Since it's been a while and this still isn't resolved, how about adding a note about this to the README.md?

The README makes it look like the packages are still available, and the linked [Microsoft documentation](https://learn.microsoft.com/en-ca/sql/tools/sqlcmd/sqlcmd-utility?view=sql-server-ver16&tabs=go%2Clinux&pivots=cs1-bash) also makes no mention of the packages being unavailable. It would have saved me a bit of wasted time if this announcement was more visible in the documentation.
